app/services/NextUpdatesService.scala

```diff
@@ -92,6 +97,15 @@ class NextUpdatesService @Inject()(val obligationsConnector: ObligationsConnecto
 
   }
 
+  def getSubmissionCounts(optOutTaxYear: TaxYear)
+                         (implicit hc: HeaderCarrier, mtdUser: MtdItUser[_]): Future[(Int, Int)]  = {
```

Comment:
`[Int, Int]` seems to be ambiguous. can you make the `[TaxYear, Int]`

Reply:
done